x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/comments.cy.ts

```diff
@@ -1,237 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import type { ExceptionListTypeEnum, NamespaceType } from '@kbn/securitysolution-io-ts-list-types';
-import { getException, getExceptionList } from '../../../objects/exception';
-import { getNewRule } from '../../../objects/rule';
-
-import { createRule } from '../../../tasks/api_calls/rules';
-import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
-
-import { login, visitWithoutDateRange } from '../../../tasks/login';
-import {
-  addExceptionFlyoutFromViewerHeader,
-  goToEndpointExceptionsTab,
-  goToExceptionsTab,
-  openEditException,
-  openExceptionFlyoutFromEmptyViewerPrompt,
-} from '../../../tasks/rule_details';
-import {
-  addExceptionComment,
-  addExceptionConditions,
-  addExceptionFlyoutItemName,
-  clickCopyCommentToClipboard,
-  submitEditedExceptionItem,
-  submitNewExceptionItem,
-  clickOnShowComments,
-  selectOs,
-} from '../../../tasks/exceptions';
-import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
-import {
-  EXCEPTION_ITEM_VIEWER_CONTAINER,
-  EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT,
-  LOADING_SPINNER,
-} from '../../../screens/exceptions';
-import {
-  createEndpointExceptionList,
-  createExceptionList,
-  createExceptionListItem,
-} from '../../../tasks/api_calls/exceptions';
-import { ROLES } from '../../../../common/test';
-
-interface ResponseType {
-  body: {
-    id: string;
-    list_id: string;
-    type: ExceptionListTypeEnum;
-    namespace_type: NamespaceType;
-  };
-}
-describe('Add, copy comments in different exceptions type and validate sharing them between users', () => {
```

Comment:
Resort to only one user, should fix this issue? Is that correct? From [observations](https://github.com/elastic/kibana/issues/162565#issuecomment-1653410937)
> Navigating to Exceptions from the Rules table after logging in with the second user

That leaves us with: 
> Asserting that the exception table has two exceptions because the created exception doesn't appear immediately in the
table after the addition, as well asserting that comment is created successfully

Maybe page reload/retry might help with it?

Reply:
Thanks, Vitalii for the review :) 

> I would advocate for leaving at least one UI test for comments

We have UI coverage for the comments panel in Cypress in other tests one of them `x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/rule_exceptions/auto_populate_with_alert_data.cy.ts`


Unfortunately, the test sometimes is quite flaky in the first steps because the [Exception flyout keeps re-rendering](https://github.com/elastic/kibana/issues/154994) which was the reason behind skipping the Exception Flyout validation suite as well.

Reply:
However, I agree with you that it is not clear in the issue that the issue is not related to the logic.

I will update it with the issue related to multiple rendering

Reply:
@WafaaNasr 
> pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/rule_exceptions/auto_populate_with_alert_data.cy.ts

is there tested functionality related to comments? I don't see any calls for `addExceptionComment`. It looks like we don't have any tests that cover this part of UI anymore